### DISCUSSION TITLE
IRhvac {"Vendor":"Trotec", "Power":"On"} not working for Trotec PAC3550 Pro

### DISCUSSION THREAD

**Discussion by cool-lox** [arendst/Tasmota#12471] [Q&A] [answered]
Hello I`trying to use above IRhvac code for my Trotec AC PAC3550 which is not working I saw in code mainly with IRhvac code only PAC 3200 A/C is supported. Any chance to implement also PAC 3550 Pro Model?
I tryed too option Model:1 or others. I`m using YTF IR Bridge Module with Tasmota 9.0.0.2(ir).

As workaround using this:
I can start AC Cooling 18° with:
IRSEND 0,+12005-5130+545-500+600-1950+540d+595fCdHfCdHf+550-495CjHfCdCdGdEf+590fCdGdCdCjCdCdCdGdCjCdCdCdGdHf+520lHfCdGdC-525+515dClLmLdGmLdGdCdGdClMjLlCd+570-475G-505KfHfCdGdO-470HfO-1975CdCdClOfIjCdCjOrIjHfOsCmOfHf+565pC

And Power off with:
IRSEND 0,+12005-5130+545-525+600-1940CcE-1915+550cE-1945CdEiCcCdEiCcCdCdCdE-1950CcCdC-495HcCkHkHcCdCdCdCdCdCk+630iC-530EfCdCkHdCcCdCdCcCdCkHkHcCcCcCkHdCkHcCdEjEiCkHcCdE-1920LiCcCcCdEjCkHdCcCcCdEiEiCd+605fCdCdC

For basic start/stop is o.k. but would be nice have more options with IRhvac. Orignial remote control is available.

Any suggestions ?

BR
Boris

**Comment by Jason2866** [comment, marked as answer]
Tasmota uses the IRremoteESP8266 lib from https://github.com/crankyoldgit/IRremoteESP8266
for all IR protocols. To get support for your model you have to ask there.

**Comment by cool-lox** [comment]
O.k. thanks will do.
